examples/direct-upload-to-gcs/appengine/main.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2019 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"github.com/google/uuid"
+	"golang.org/x/oauth2/google"
+	iam "google.golang.org/api/iam/v1"
+)
+
+var (
+	// iamService is a client for calling the signBlob API.
+	iamService *iam.Service
+
+	// serviceAccountName represents Service Account Name.
+	// See more details: https://cloud.google.com/iam/docs/service-accounts
+	serviceAccountName string
+
+	// serviceAccountID follows the below format.
+	// "projects/%s/serviceAccounts/%s"
+	serviceAccountID string
+
+	// uploadableBucket is the destination bucket.
+	// All users will upload files directly to this bucket by using generated Signed URL.
+	uploadableBucket string
+)
+
+func init() {
```

Comment:
This code is not an importable package, which is generally the only place an init() func is required. The code here should go into main().

Reply:
done. it was old appengine classic style :(